src/Filter.php

```diff
@@ -91,6 +98,23 @@ public function getIgnored(): array
         return $this->ignored->toArray();
     }
 
+    public function default($value): self
+    {
+        $this->default = $value;
+
+        return $this;
+    }
+
+    public function getDefault()
+    {
+        return $this->default;
+    }
+
+    public function defaultSet(): bool
```

Comment:
Maybe you could add a verb to the method name, to better fit in with other "check" methods: `isDefaultSet()`

Reply:
Done.